Packages/PrimitivesComponents/Sources/Types/TransactionHeaderInputType.swift

```diff
@@ -4,7 +4,72 @@ import Foundation
 import Primitives
 import BigInt
 
-public enum TransacitonHeaderInputType: Sendable {
+public enum TransactionHeaderInputType: Sendable {
+    public struct SwapContext: Sendable {
```

Comment:
```suggestion
    public struct SwapContext: Sendable {
```

1) move it out of `TransactionHeaderInputType`
2) Improve naming, this is swap metadata, context naming is not relevant here
3)